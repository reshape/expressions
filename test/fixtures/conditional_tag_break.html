<if condition='true'>
  <p>if</p>
</if>
<p>x</p>
<else>
  <p>else</p>
</else>
