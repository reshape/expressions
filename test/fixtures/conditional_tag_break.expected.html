  <p>if</p>
<p>x</p>
<else>
  <p>else</p>
</else>
